e2e/browser/browser.test.js

```diff
@@ -57,19 +57,19 @@ describe("E2E browser", () => {
     await gammeDriver.stop();
   });
 
-  it("should create Orbs.Client instance", async () => {
-    await clickOnElement("#create-orbs-client");
-    const orbsClientResult = await getElementText("#orbs-client-result");
-    expect(orbsClientResult).toEqual("Created");
-  });
-
   it("should create the sender account", async () => {
     await clickOnElement("#create-sender-account");
     const senderAccountId = await getElementText("#sender-account-id");
     const accountIdLength = senderAccountId.split(",").length;
     expect(accountIdLength).toEqual(32);
   });
 
+  it("should create Orbs.Client instance", async () => {
```

Comment:
These lines were just moved, right?

Reply:
Yeah. I just moved to async/await syntax.